### PR TITLE
French translation 3.8.0

### PR DESCRIPTION
**Pull request by NorXAengell**
Some translation/correction for the 3.8.0

**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Respect the [Submitting Content Guidelines](https://github.com/CBATeam/CBA_A3/wiki/Submitting%20content)

### CODE REVIEW

File: addons/accessory/stringtable.xml [hunk 1]
```diff
@@ -16,6 +17,7 @@
             <Chinesesimp>下个导轨物品状态</Chinesesimp>
             <Russian>Следующее состояние приспособления на оружии</Russian>
             <Polish>Następny tryb akcesoria na szynie</Polish>
+            <French>Mode de l'accessoire sur rail suivant </French>
```
Comment (bux) [inline addons/accessory/stringtable.xml]: space too much

File: addons/accessory/stringtable.xml [hunk 2]
```diff
@@ -32,6 +35,7 @@
             <Chinesesimp>上个导轨物品状态</Chinesesimp>
             <Russian>Предыдущее состояние приспособления на планке</Russian>
             <Polish>Poprzedni tryb akcesoria na szynie</Polish>
+            <French>Mode de l'accessoire sur rail précèdent </French>
```
Comment (bux) [inline addons/accessory/stringtable.xml]: space too much

File: addons/accessory/stringtable.xml [hunk 3]
```diff
@@ -48,6 +53,7 @@
             <Chinesesimp>下个瞄镜模式</Chinesesimp>
             <Russian>Следующий режим оптики</Russian>
             <Polish>Następny tryb optyki</Polish>
+            <French>Mode de l'optique suivant </French>
```
Comment (bux) [inline addons/accessory/stringtable.xml]: space too much

File: addons/accessory/stringtable.xml [hunk 4]
```diff
@@ -64,6 +71,7 @@
             <Chinesesimp>上个瞄镜模式</Chinesesimp>
             <Russian>Предыдущее состояние оптики</Russian>
             <Polish>Poprzedni tryb optyki</Polish>
+            <French>Mode de l'optique sur rail précèdent </French>
```
Comment (bux) [inline addons/accessory/stringtable.xml]: space too much

File: addons/common/stringtable.xml [hunk 5]
```diff
@@ -13,6 +13,7 @@
         <Key ID="STR_cba_common_need_mission_start">
             <English>You must first start a mission.</English>
             <German>Sie müssen zuerst eine Mission starten.</German>
+            <French>Vous devez d'abord commencer une mission.</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/common/stringtable.xml]: newline at end of file missing now

File: addons/diagnostic/stringtable.xml [hunk 6]
```diff
@@ -62,6 +65,7 @@
             <Chinesesimp>[CBA] 允许远程为目标进行除错。需要开启除错控制台。</Chinesesimp>
             <Russian>[CBA] Разрешает отладку дистанционной цели. Требуется Консоль Отладки.</Russian>
             <Polish>[CBA] Włącza debugowanie zdalnego celu. Konsola Debugowania musi być dostęna.</Polish>
+            <French>[CBA] Permet le débogage de la cible distante. Nécessite la console de débogage.</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/diagnostic/stringtable.xml]: newline EOF

File: addons/hashes/stringtable.xml [hunk 7]
```diff
@@ -8,6 +8,7 @@
             <Chinesesimp>社群基础模组 - 杂凑</Chinesesimp>
             <Russian>Community Base Addons - Хэши</Russian>
             <Polish>Community Base Addons - Hashe</Polish>
+            <French>Community Base Addons - Hashes</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/hashes/stringtable.xml]: newline EOF

File: addons/help/stringtable.xml [hunk 8]
```diff
@@ -81,4 +82,4 @@
             <Chinesesimp>制作组 - 模组</Chinesesimp>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/help/stringtable.xml]: newline EOF

File: addons/jr/stringtable.xml [hunk 9]
```diff
@@ -25,4 +26,4 @@
             <Chinesesimp>机枪用消音器</Chinesesimp>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/jr/stringtable.xml]: newline EOF

File: addons/keybinding/stringtable.xml [hunk 10]
```diff
@@ -43,4 +45,4 @@
             <French>Touche inconnue %1</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/keybinding/stringtable.xml]: newline EOF

File: addons/main/stringtable.xml [hunk 11]
```diff
@@ -49,8 +49,8 @@
             <Chinese>模組:</Chinese>
             <Chinesesimp>模组:</Chinesesimp>
             <Japanese>アドオン:</Japanese>
-            <French>ADDON:</French>
+            <French>ADDON :</French>
             <Polish>ADDON:</Polish>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/main/stringtable.xml]: newline EOF

File: addons/modules/stringtable.xml [hunk 12]
```diff
@@ -656,6 +737,7 @@
             <Chinesesimp>同步到群组的队长身上，使该群组巡逻指定的地点</Chinesesimp>
             <Russian>Синхронизируйте с лидером группы, чтобы патрулировать указанное местоположение</Russian>
             <Polish>Zsynchronizuj z liderem grupy aby patrolować określoną lokalizację</Polish>
+            <French>Synchroniser avec le chef du groupe pour défendre un emplacement désigné</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/modules/stringtable.xml]: newline EOF

File: addons/settings/stringtable.xml [hunk 13]
```diff
@@ -53,7 +57,7 @@
             <Chinesesimp>模组设定选项...</Chinesesimp>
             <Portuguese>Opções ded Extensão...</Portuguese>
             <Russian>Настройки Дополнений...</Russian>
-            <French>Addon Options...</French>
+            <French>Ajuster les paramètres des addons.</French>
```
Comment (commy2) [inline addons/settings/stringtable.xml]: This is shown in 3den. May want to check if `...` is not the style used by the base game strings there in French as well.

File: addons/settings/stringtable.xml [hunk 14]
```diff
@@ -319,8 +329,8 @@
             <Chinese>顯示預設值:</Chinese>
             <Chinesesimp>显示预设值:</Chinesesimp>
             <Japanese>標準の変数を表示:</Japanese>
-            <French>Afficher les valeurs par défaut:</French>
+            <French>Afficher les valeurs par défaut :</French>
             <Polish>Pokaż domyślnę wartości:</Polish>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/settings/stringtable.xml]: newline EOF

File: addons/statemachine/stringtable.xml [hunk 15]
```diff
@@ -9,6 +9,7 @@
             <Portuguese>Extensões de Base Comunitária - Component da Máquina de Estado</Portuguese>
             <Russian>Community Base Addons - Компонент Машины Состояний</Russian>
             <Polish>Community Base Addons - Komponent Maszyny Stanów</Polish>
+            <French>Community Base Addons - Composant machine d'état</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/statemachine/stringtable.xml]: newline EOF

File: addons/strings/stringtable.xml [hunk 16]
```diff
@@ -26,17 +27,10 @@
...
-            <Spanish> </Spanish>
-            <Italian> </Italian>
             <Japanese>,</Japanese>
             <Chinese>,</Chinese>
             <Chinesesimp>,</Chinesesimp>
             <Portuguese>.</Portuguese>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/strings/stringtable.xml]: newline EOF

File: addons/strings/stringtable.xml [hunk 17]
```diff
@@ -26,17 +27,10 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
```
[the remark this answers is not in the document]
Comment (commy2) [reply addons/strings/stringtable.xml]: This. Spaces, apparently, are the decimal separators in these languages.

File: addons/versioning/stringtable.xml [hunk 18]
```diff
@@ -9,6 +9,7 @@
             <Portuguese>Extensões de Base Comunitária - Versionamento</Portuguese>
             <Russian>Community Base Addons - Версионность</Russian>
             <Polish>Community Base Addons - Wersjonowanie</Polish>
+            <French>Community Base Addons - Version</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/versioning/stringtable.xml]: newline EOF

File: addons/xeh/stringtable.xml [hunk 19]
```diff
@@ -9,6 +9,7 @@
             <Portuguese>Extensões de Base Comunitária - Manipuladores Extendidos de Eventos</Portuguese>
             <Russian>Community Base Addons - Расширенные Обработчики Событий</Russian>
             <Polish>Community Base Addons - Rozszerzona Obsługa Zdarzeń</Polish>
+            <French>Community Base Addons - Gestionnaires d'évènement étendus</French>
         </Key>
     </Package>
-</Project>
+</Project>
```
Comment (commy2) [inline addons/xeh/stringtable.xml]: newline EOF

**Comment by commy2** [comment]
Thanks @NorXAengell! If you don't want to yourself, I can fix the comments myself next weekend.

**Comment by NorXAengell** [comment]
Thx commy2; its not that i don't want to do it my self, it's just that i don't know how to do this "well" and i will certainly broke things more -_-...So if you could do this it will be kind.
Sorry for errors and for the EOF.

**Pull request closed without merging**